openpype/settings/defaults/project_settings/hiero.json

```diff
@@ -1,4 +1,32 @@
 {
+    "ext_mapping": {
+        "model": "hrox",
+        "mayaAscii": "hrox",
+        "camera": "hrox",
+        "rig": "hrox",
+        "workfile": "hrox",
+        "yetiRig": "hrox"
+    },
+    "hiero-dirmap": {
+        "enabled": false,
+        "paths": {
+            "source-path": [],
+            "destination-path": []
+        }
+    },
```

Comment:
There are not schemas for these settings. Also you've created defaults manually, please use settings UI in develop mode to create default settings.
```suggestion
```

### How to run settings UI in develop mode
There is a script `openpype/tools/run_settings`. If you run it you should have available `Modify defaults` checkbox. From that moment if you save settings they will change default settings in json file.
![image](https://user-images.githubusercontent.com/43494761/177146938-3433d218-96bd-43f5-90e0-5f3fe189e1b5.png)